### PR TITLE
sling12/Issues/448: make sure we dont have large external dependencies

### PR DESCRIPTION
**Pull request by flx-sta**
pr redo after messing up #656

### CODE REVIEW

**Comment by Felix-Puetz** [comment]
Review
@reusr1 
### After Login
image
* I don't think we should be loading images of the CMS itself from an external source (headwire.com)
* We should use local versions of the css for `font-awesome` and `ionicons`

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html [hunk 1]
```diff
@@ -113,7 +113,6 @@
 </sly>
 
 <sly data-sly-include="renderer.html"></sly>
-<script src="https://unpkg.com/vue-codemirror-lite/dist/vuecodemirror.min.js"></script>
```
Comment (flx-sta) [inline admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html:116]: This worked 👍

**Review by flx-sta — COMMENTED** [review]
@reusr1 
### After Login
image
* I don't think we should be loading images of the CMS itself from an external source (headwire.com)
* We should use local versions of the css for `font-awesome` and `ionicons`
otherwise offline the rich-toolbar looks like this
image


### On Edit Page
image
* You did mention this though so I'll leave it to you if you want to keep it like this or want to download these too

**Comment by flx-sta** [comment]
~~@reusr1 please review the changes i made since i can't review it myself... except waiting for a new docker image~~

nevermind it is broken

**Comment by flx-sta** [comment]
font-awesome whole module is now exported into the felibs directory due to dependencies of fonts

**Pull request merged**